src/app/script/security.cpp

```diff
@@ -93,7 +94,7 @@ std::string get_script_filename(lua_State* L)
   const char* source = lua_tostring(L, -1);
   std::string script;
   if (source && *source)
-    script = source+1;
+    script = source;
```

Comment:
About your question:

> It was also saying "nternal" because the text was getting +1'd for some reason? Unclear why that was there and the git blame was not particularly illuminating since it's always been like this.

The `source` field contains a `"@"` at the beginning for files, e.g.

Create a `test.lua` in your `scripts` folder:
```lua
print(debug.getinfo(1, "S").source)
```

And it will print:
```
@/home/david/.config/aseprite/scripts/test.lua
```

We need this +1 to get the proper filename.